pallets/frequency-tx-payment/src/tests/pallet_tests.rs

```diff
@@ -1,5 +1,7 @@
-use super::*;
-use crate::{self as pallet_frequency_tx_payment, mock::*, ChargeFrqTransactionPayment};
+use crate::{
```

Comment:
With @rlaferla's test refactor, we decided to organize tests by function and not location, but this can be in a follow-up.

Reply:
Agreed. This was more for test coverage calculations which needs things under /test or it picks up extra stuff